firmware/tools/export_external_apps.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2023 Bernd Herzog
+#
+# This file is part of PortaPack.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2, or (at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; see the file COPYING.  If not, write to
+# the Free Software Foundation, Inc., 51 Franklin Street,
+# Boston, MA 02110-1301, USA.
+#
+
+import os
+import sys
+import struct
+import subprocess
+
+usage_message = """
+PortaPack external app image creator
+
+Usage: <command> <project source dir> <binary dir> <cmake objcopy path> <list of external image prefixes>
+"""
```

Comment:
nit: what's that list look like? Maybe add a sample command line in there so it's clear how to pass multiple items.

Reply:
clarified the comment